lib/network_utils.pm

```diff
@@ -65,6 +65,11 @@ Main use case is post_fail_hook, to be able to upload logs
 sub recover_network {
     my (%args) = @_;
 
+    if ($args{reset_network}) {
+        script_run('rm /etc/sysconfig/network/ifcfg-*');
+        script_run('rcwickedd restart');
+        script_run('rcwicked restart');
```

Comment:
Do we need to check for SLES or something, so we know we are running wicked not network-manager?

Reply:
Or we just move this inside of `upload_wicked_logs()`

Reply:
1. I really think that it is right place for this logic and don't want to move it in some other place 
2. I don't want to introduce extra checks because it is not default flow of this function but some extra param which you need to explicitly specify to make it work 
3. So responsibility of misuse on person who used parameter and forgot to read documentation which I updated

Reply:
ok